app/src/main/java/com/zeapo/pwdstore/git/BaseGitActivity.kt

```diff
@@ -109,6 +116,14 @@ abstract class BaseGitActivity : AppCompatActivity() {
                 result
             }
         }
+        if (hostname != previousHostname) {
+            encryptedSettings.edit().apply {
+                when (protocol) {
+                    Protocol.Ssh -> remove("ssh_key_passphrase")
```

Comment:
The name of the setting may be a bit confusing here: Are we storing the passphrase for a local SSH key (in which case we wouldn't need/want to wipe it) or is this a passphrase used for SSH password authentication?

Reply:
It's the passphrase that allows us to decrypt our local key. Might not make sense to wipe it after all. Suggestions for a better name?

Reply:
> It's the passphrase that allows us to decrypt our local key. Might not make sense to wipe it after all. Suggestions for a better name?

Sorry, I missed your reply earlier. Maybe "ssh_key_local_passphrase" to clarify that this is not sent to the server? We shouldn't wipe it, we want people to reuse their SSH key pairs after all ;-)

Reply:
Fixed with 95ec00929230aa4bcc12ef5f4e367f5e1307fa68